packages/css/src/components/navbar/navbar.scss

```diff
@@ -4,6 +4,54 @@ $dso-navigation-selected-color: $mauve;
   border-bottom: 4px solid $color;
 }
 
+.dso-navbar {
+  .dso-navbar-header {
+    float: left;
+  }
+
+  @media screen and (max-width: $screen-sm-max) {
+    clear: left;
+
+    > .dso-nav {
+      clear: both;
+    }
+
+    &:not(.dso-open) {
+      > .dso-nav {
+        display: none;
+
+        &.dso-nav-sub {
+          display: block;
+        }
+      }
+    }
+  }
+
+  &.dso-navbar {
```

Comment:
hier snap ik niks van. krijg je nu `.dso-navbar.dso-navbar` ?

Reply:
Volgens mij had dit ooit te maken met "css gewicht", maar ik denk dat we dat station nu al gepasseerd zijn. Als ik dit ombouw naar "normaal", dan ziet het er nog hetzelfde uit,